css/style.css

```diff
@@ -24,12 +24,74 @@ html {
 /*******************************************/
 
 body {
-  color: #000;
+  color: #033048;
+  Background-color: white;
   font-size: 12px;
   line-height: 1.4;
-  font-family: Helvetica, Arial, sans-serif;
+  font-family: 'Open Sans', sans-serif;
+  margin-left: 50px;
+  margin-right: 50px;
 }
 
+h1 {
+  color: #033048;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 40px;
+  font-weight: 600px;
```

Comment:
font-weight is expressed as a unitless value, so it should be font-weight: 600; (no px at the end)